src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -660,11 +660,9 @@ internal static object InferTypeAndConvertLiteral(string literal)
                 {
```

Comment:
(not directly related): This check can be removed, it duplicates the check at line 638.